### DISCUSSION TITLE
MobaXterm_Portable_v24.2_chs

### DISCUSSION THREAD

**Discussion by RipplePiam** [RipplePiam/MobaXterm-Chinese-Simplified#61] [General]
**🎉🎉🎉 MobaXterm Portable v24.2 (2024-07-02) 汉化版发布啦 !**

**更新说明：**

- 本次更新可解决 #43 #47 #49 #50 #51 #59 #60
- 修复数个安全漏洞
- 提供更多高级设置
- 软件优化

**软件更新日志（来自官方网站）：**
- [更新日志.txt](https://github.com/user-attachments/files/16757782/default.txt)

<hr /><em>This discussion was created from the release <a href='https://github.com/RipplePiam/MobaXterm-Chinese-Simplified/releases/tag/v0.24.2.96'>MobaXterm_Portable_v24.2_chs</a>.</em>

**Comment by wy16W2pIilK1xgqN** [comment]
非常感谢。
英盲大救星🥰

**Comment by myh66** [comment]
爱死作者了🥰

**Comment by amxoy** [comment]
幸福来的太突然了，感谢大佬

**Comment by YBxbxxz** [comment]
好好好，感谢作者
